sycl/test/abi/layout_handler.cpp

```diff
@@ -1,6 +1,8 @@
 // RUN: %clangxx -fsycl -c -fno-color-diagnostics -Xclang -fdump-record-layouts %s | FileCheck %s
 // REQUIRES: linux
 
+// clang-format off
```

Comment:
I suggest disable clang-format check only for CHECK comment, but keep it ON for the test code.

Reply:
Done